libcudacxx/test/libcudacxx/cuda/atomics/atomic.uninitialized.pass.cpp

```diff
@@ -0,0 +1,77 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// UNSUPPORTED: libcpp-has-no-threads, pre-sm-60
+// UNSUPPORTED: windows && pre-sm-70
+// UNSUPPORTED: nvrtc
+// ADDITIONAL_COMPILE_OPTIONS_CUDA: --maxreccount 24
```

Comment:
Is this a typo? https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/#maxrregcount-amount-maxrregcount
```suggestion
// ADDITIONAL_COMPILE_OPTIONS_CUDA: --maxrregcount 24
```

Reply:
It was and I also found that our lit machinery does not properly work that deep in the pipeline so I had to do some surgery